kubernetes/samples/charts/traefik/README.md

```diff
@@ -4,102 +4,104 @@ This sample demonstrates how to install the Traefik ingress controller to provid
-The chart is in the default repository for Helm.
+This document is based on Traefik version 2.x with the Helm chart at [Traefik Helm Repository](https://github.com/containous/traefik-helm-chart).
+For more information about Traefik, see [Traefik Official Site](https://traefik.io/) 
 
 To install the Traefik operator in the `traefik` namespace with default settings:
 ```
-$ helm install --name traefik-operator --namespace traefik stable/traefik
+$ helm repo add traefik https://containous.github.io/traefik-helm-chart
+$ helm repo update
+$ kubectl create namespace traefik
+$ helm install traefik-operator traefik/traefik --namespace traefik
 ```
-Or, with a given `values.yaml`:
+You can also install the Traefik operator with a custom `values.yaml`. For more detailed information, see [Traefik GitHub Project](https://github.com/containous/traefik-helm-chart/blob/master/traefik/values.yaml).
 ```
-$ helm install --name traefik-operator --namespace traefik --values values.yaml stable/traefik
-```
-With the dashboard enabled, you can access the Traefik dashboard with the URL `http://${HOSTNAME}:30305`, with the HTTP host `traefik.example.com`.
-```
-$ curl -H 'host: traefik.example.com' http://${HOSTNAME}:30305/
-```
-
-## Optionally, download the Traefik Helm chart
-If you want, you can download the Traefik Helm chart and untar it into a local folder:
-```
-$ helm fetch  stable/traefik --untar
-```
-
-## Update the Traefik operator
-After the Traefik operator is installed and running, if you want to change some configurations of the operator, use `helm upgrade` to achieve this.
-```
-$ helm upgrade traefik-operator stable/traefik --values values.yaml 
+$ helm install traefik-operator traefik/traefik --namespace traefik --values values.yaml
 ```
 
-## Configure Traefik as a load balancer for WLS domains
-In this section we'll demonstrate how to use Traefik to handle traffic to backend WLS domains.
+## Configure Traefik as a load balancer for WebLogic domains
+This section describes how to use Traefik to handle traffic to backend WebLogic domains.
 
-### 1. Install WLS domains
-Now we need to prepare some domains for Traefik load balancing.
+### 1. Install WebLogic domains
+First, we need to prepare two domains for Traefik load balancing.
 
-Create two WLS domains:
-- One domain with name `domain1` under namespace `default`.
-- One domain with name `domain2` under namespace `test1`.
+Create two WebLogic domains:
+- One domain with `domain1` as domain uid under namespace `weblogic-domain1`.
+- One domain with `domain2` as domain uid under namespace `weblogic-domain2`.
 - Each domain has a web application installed with the URL context `testwebapp`.
+- Each domain has a WebLogic cluster `cluster-1` where each managed server listens on port `8001`.
 
-### 2. Install the Traefik Ingress
-#### Install a host-routing Ingress
+### 2. Web request routing 
+#### Host based routing 
```

Comment:
Host based routing  -> Host-based routing (lets hyphenate the different routing types)

Reply:
I don't see this change: Host based routing -> Host-based routing (hyphenated)